browser-test/src/applicant/questions/dropdown.test.ts

```diff
@@ -66,7 +66,7 @@ test.describe(
           },
           /* clickSubmit= */ false,
         )
-        await adminQuestions.expectPreviewOptions(['Sample question option'])
+        await adminQuestions.expectPreviewOptions(['Sample question option\n'])
```

Comment:
(question) Are there any cases where the `\n` would affect an element's `id` or `name`?